app/views/users/index.html.erb

```diff
@@ -13,11 +13,9 @@
 
 <% @users.each do |user| %>
   <tr>
-    <td><%= user.name %></td>
+    <td><%= link_to user.name, user_path(user) %></td>
     <td><%= user.email %></td>
-    <td><%= link_to 'Show', user_path(user) %></td>
-    <td><%= link_to 'Edit', edit_user_path(user) %></td>
```

Comment:
Disagree.  I've routinely clicked directly on the Edit link, because it saves me clicks.  (Most often, it's Ctrl+F, type in the user's name, press Tab to get me to the Edit link, and press Enter to immediately edit the user.)  I don't care if there are multiple links to the same Edit page; in fact, I think it often helps, because they're just shortcuts getting me directly where I need to go.

Cleanliness and simplicity are fine; making things take longer to accomplish merely in the name of simplicity is a false savings.